src/Assets/frequencyOptions.tsx

```diff
@@ -3,6 +3,7 @@ import { FormattedMessage } from 'react-intl';
 const frequencyOptions = {
   weekly: <FormattedMessage id="frequencyOptions.weekly" defaultMessage="every week" />,
   biweekly: <FormattedMessage id="frequencyOptions.biweekly" defaultMessage="every 2 weeks" />,
+  twiceMonth: <FormattedMessage id="frequencyOptions.twiceMonth" defaultMessage="twice a month" />,
```

Comment:
The twice a month option in the backend is named `semimonthly`. Please name this the same in here as well.